<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>389</width>
          <height>481</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>389</width>
          <height>481</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <property name="spacing">
            <number>30</number>
           </property>
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>281</width>
               <height>41</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Yu Gothic UI Semibold</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: white;
border-style:solid; 
border-width: 0px; 
color: rgb(27, 43, 66);
</string>
             </property>
             <property name="text">
              <string>Register to the account</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="RLoginLEdit">
             <property name="minimumSize">
              <size>
               <width>310</width>
               <height>44</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>310</width>
               <height>44</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Yu Gothic Medium</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: white;
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="RPasswordLEdit">
             <property name="minimumSize">
              <size>
               <width>310</width>
               <height>44</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>310</width>
               <height>44</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Yu Gothic Medium</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: white;
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <property name="leftMargin">
            <number>130</number>
           </property>
           <property name="rightMargin">
            <number>130</number>
           </property>
           <item>
            <widget class="QPushButton" name="RRegisterButton">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>44</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>44</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Yu Gothic Medium</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Register</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
